lab_1_keywords_tfidf/main.py

```diff
@@ -33,7 +41,12 @@ def remove_stop_words(tokens: list[str], stop_words: list[str]) -> Optional[list
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    for i in tokens[::-1]:
```

Comment:
why do you loop over reverse tokens?

Reply:
done by appending correct words to an empty list